src/pages/patterns/filtering/index.mdx

```diff
@@ -69,6 +69,62 @@ Multiple category selection is most commonly manifested in either a vertical lis
 </Column>
 </Row>
 
+### Advanced filtering
+
+_Advanced search_ can be used when the user requires to search and filter a list of items by choosing more than one filter from multiple filter categories. This pattern is especially useful when the number of items within each category is dynamic, such as filtering a table by a user or user defined tags.
+
+Advanced search is also commonly used when there is not the requried amount of space on the screen to accommodate a vertical list.
+
+<br />
+<Row>
+<Column colLg={8}>
+
+![Contextual image](images/filter-8.png)
```

Comment:
For alt text, please use more detailed descriptions of what's in the image. It doesn't have to be long, but "Contextual image" doesn't provide much utility to a visually impaired reader using a screen reader.